mergify_engine/engine/__init__.py

```diff
@@ -145,6 +177,14 @@ async def _ensure_summary_on_head_sha(ctxt: context.Context) -> None:
         return
 
     previous_summary = await _get_summary_from_sha(ctxt, sha)
+
+    if previous_summary is None:
+        # NOTE(sileht): If the cached summary sha expires and the next event we got for
+        # a pull request is "synchronize" we will loose the summary. Most of the times
+        # it's not a big deal, but if the pull request in queued for merge, it may
+        # be stuck.
```

Comment:
```suggestion
        # NOTE(sileht): If the cached summary sha expires and the next event we got for
        # a pull request is "synchronize" we will lose the summary. Most of the times
        # it's not a big deal, but if the pull request is queued for merge, it may
        # be stuck.
```